week-1/2-mandatory/3-function-output.js

```diff
@@ -1,16 +1,22 @@
 // Add comments to explain what this function does. You're meant to use Google!
+// This function generate a random number between a range of 0 to 10.
+// Math.random() generate a floating point number between a range of 0 to 1. 
+// In which 0 is inclusive and 1 is exclusive.
 function getNumber() {
   return Math.random() * 10;
 }
 
 // Add comments to explain what this function does. You're meant to use Google!
+// concat() method is used to combine or join two strings.
+// In this function w2 is added to the infront of w1.
 function s(w1, w2) {
   return w1.concat(w2);
 }
 
 function concatenate(firstWord, secondWord, thirdWord) {
   // Write the body of this function to concatenate three words together
   // Look at the test case below to understand what to expect in return
+  return firstWord+" "+secondWord+" "+thirdWord
```

Comment:
This definitely works but could you think of another way of doing this, perhaps using `concat`?